e2e/fragments/addressPostcodeLookup.js

```diff
@@ -0,0 +1,38 @@
+/*global locate*/
+
+const I = actor();
+
+module.exports = {
+
+  fields: {
+    postcodeLookup: '#postcodeInput',
+    addressList: '#addressList',
+    buildingAndStreet: {
+      addressLineOne: '#AddressLine1',
+      addressLineTwo: '#AddressLine2',
+      addressLineThree: '#AddressLine3',
+    },
+    town: '#PostTown',
+    county: '#County',
+    postcodeZipcode: '#PostCode',
+    country: '#Country',
+  },
+  findAddress: 'Find address',
+  cantEnterPostcode: locate('span').withText('I can\'t enter a UK postcode'),
+  
+  lookupPostcode({address}) {
```

Comment:
Why is the argument in curly brackets?

Reply:
I'm de-structuring an object to just get address. This will be changed to just address as per Sam's comment.